fractions/io/src/main/java/org/wildfly/swarm/io/IOFraction.java

```diff
@@ -30,7 +30,11 @@
 public class IOFraction extends IO<IOFraction> implements Fraction<IOFraction> {
 
     public IOFraction applyDefaults() {
-        return worker(new Worker("default"))
+        return worker(
+                "default", w -> {
+                    w.ioThreads(100);
+                    w.taskMaxThreads(20);
+                })
                 .bufferPool(new BufferPool("default"));
```

Comment:
Not worth editing, but `.bufferPool("default")` would also suffice and delegate to the factory internally.